src/SablierMerkleFactory.sol

```diff
@@ -135,15 +137,12 @@ contract SablierMerkleFactory is
             abi.encodePacked(msg.sender, abi.encode(baseParams), lockup, cancelable, transferable, abi.encode(schedule))
         );
 
-        // Compute the fee for the user.
-        uint256 fee = _computeFeeForUser(msg.sender);
-
         // Deploy the MerkleLL contract with CREATE2.
-        merkleLL = new SablierMerkleLL{ salt: salt }(baseParams, lockup, cancelable, transferable, schedule, fee);
+        merkleLL = new SablierMerkleLL{ salt: salt }(msg.sender, baseParams, lockup, cancelable, transferable, schedule);
```

Comment:
Named parameters

Reply:
It will make it fit into multiple lines. Except for `msg.sender`, all the variables have the same name (baseParams will have params). Thats why I didn't use named parameters here but will do now.

Reply:
I don't have an issue with multiple lines. Small cost to pay for extra clarity.